pkg/config/user_config.go

```diff
@@ -73,17 +73,17 @@ type GuiConfig struct {
 	SkipRewordInEditorWarning bool `yaml:"skipRewordInEditorWarning"`
 	// Fraction of the total screen width to use for the left side section. You may want to pick a small number (e.g. 0.2) if you're using a narrow screen, so that you can see more of the main section.
 	// Number from 0 to 1.0.
-	SidePanelWidth float64 `yaml:"sidePanelWidth"`
+	SidePanelWidth float64 `yaml:"sidePanelWidth" jsonschema:"maximum=1,minimum=0"`
 	// If true, increase the height of the focused side window; creating an accordion effect.
 	ExpandFocusedSidePanel bool `yaml:"expandFocusedSidePanel"`
 	// Sometimes the main window is split in two (e.g. when the selected file has both staged and unstaged changes). This setting controls how the two sections are split.
 	// Options are:
 	// - 'horizontal': split the window horizontally
 	// - 'vertical': split the window vertically
 	// - 'flexible': (default) split the window horizontally if the window is wide enough, otherwise split vertically
-	MainPanelSplitMode string `yaml:"mainPanelSplitMode"`
+	MainPanelSplitMode string `yaml:"mainPanelSplitMode" jsonschema:"enum=horizontal,enum=flexible,enum=vertical"`
 	// One of 'auto' (default) | 'en' | 'zh-CN' | 'zh-TW' | 'pl' | 'nl' | 'ja' | 'ko' | 'ru'
-	Language string `yaml:"language"`
+	Language string `yaml:"language" jsonschema:"enum=auto,enum=en,enum=zh,enum=pl,enum=nl,enum=ja,enum=ko,enum=ru"`
```

Comment:
`enum=zh` seems to be wrong, this should be `enum=zh-CN,enum=zh-TW`.

Reply:
fixed in 34b0f495e4f784b5918dc0fd2170dcd1c299fec1